<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginScreen</class>
 <widget class="QMainWindow" name="loginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginScreen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>398</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
font: 14pt ;
color:#676767;
border: 1px solid #ffffff;
border-radius: 10px;</string>
    </property>
    <property name="maxLength">
     <number>500</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>   Please enter account name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdLineEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>468</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
font: 14pt;
color:#676767;
border: 1px solid #ffffff;
border-radius: 10px;</string>
    </property>
    <property name="maxLength">
     <number>500</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>   Please enter password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frogotPwdPb">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>538</y>
      <width>195</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font:  12pt ;
border: 1px solid #ffffff;
border-radius: 0px;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Forgot passward</string>
    </property>
   </widget>
   <widget class="QLabel" name="accAppLabel">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>538</y>
      <width>195</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font:  12pt ;
border: 1px solid #ffffff;
border-radius: 0px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Account application</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loginPb">
    <property name="geometry">
     <rect>
      <x>897</x>
      <y>608</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3,36,168);
font:  13pt ;
color: rgb(255,255,255);
border: 1px solid #051e7e;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="headerLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/meril_logo.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>0</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>0</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>2025/03/24</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>50</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>20:03:41</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="nameIconLable">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>398</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border: 1px solid #ffffff;
border-top-left-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/userIcon_48.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pwdIconLabel">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>468</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border: 1px solid #ffffff;
border-top-left-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/pwdIcon_50.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queMrakIconLabel">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>544</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255,255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/queMarkIcon_50.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
